src/components/common/SummaryDefineCompany.vue

```diff
@@ -5,48 +5,62 @@
         v-if="invalidSection"
         class="defineCompanyStepErrorMessage"
       >
-        <span>
```

Comment:
I deleted the useless span around this code, but that forced me to add spans around the router link text.